scheduleapi.py

```diff
@@ -0,0 +1,43 @@
+import json
+from flask import request
+from flask_restful import Resource, abort
+
+from schedule import Schedule
+
+def read_args(request):
+    return json.loads(request.data.decode('utf-8'))
+
+class ScheduleAPI(Resource):
+    def get(self, owner):
+        sched = Schedule.get(owner)
+        if sched is None:
+            abort(404, message='No schedule with owner {} exists'.format(owner))
+        return vars(sched)
+
+    def put(self, owner):
+        data = read_args(request)
+
+        sched = Schedule.get(owner)
+        if sched is None:
+            sched = Schedule({'owner': owner})
```

Comment:
instead of `owner` think of `owner_id` .. have the DAO handle the logic of HOW to update a schedule